Doc/whatsnew/3.13.rst

```diff
@@ -65,7 +65,12 @@ Summary -- Release highlights
 
 .. PEP-sized items next.
 
+* :pep:`602` ("Annual Release Cycle for Python") has been updated:
 
+  * Python 3.9 - 3.12 have 1½ year of full support,
+    followed by 3½ years of security fixes.
```

Comment:
I would prefer text in english rather than Unicode characters. I'm not sure that it's easy to read for everybody.

Reply:
Updated.